srml/generic-asset/src/lib.rs

```diff
@@ -0,0 +1,1241 @@
+// Copyright (C) 2019 Centrality Investments Limited
```

Comment:
Since a lot (most?) of the code is lifted straight from our assets and balances module, I'd expect to see a Parity copyright here, too.